android/gradle.properties

```diff
@@ -6,7 +6,7 @@
 # http://www.gradle.org/docs/current/userguide/build_environment.html
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx256m -XX:MaxPermSize=256m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx320m -XX:MaxPermSize=320m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
This config could go to ignored local gradle settings.

Reply:
now it's used to keep also project specific properties

Reply:
I was thinking of some local overrides, but docs suggest keeping JVM args public anyway: https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties
Feel free to resolve.